agent/agent.go

```diff
@@ -2007,14 +2005,47 @@ func newAPIConnRoutineManager(gracefulCtx, hardCtx context.Context, logger slog.
 	stopCtx := eitherContext(remainCtx, gracefulCtx)
 	return &apiConnRoutineManager{
 		logger:    logger,
-		conn:      conn,
+		aAPI:      aAPI,
+		tAPI:      tAPI,
 		eg:        eg,
 		stopCtx:   stopCtx,
 		remainCtx: remainCtx,
 	}
 }
 
-func (a *apiConnRoutineManager) start(name string, b gracefulShutdownBehavior, f func(context.Context, drpc.Conn) error) {
+func (a *apiConnRoutineManager) startAgentAPI(name string, b gracefulShutdownBehavior, f func(context.Context, proto.DRPCAgentClient23) error) {
```

Comment:
What's the difference between this and `startTailnetAPI`? These look like identical functions to me. 🤔

Edit: Now I see it, it's what's passed to the func (aAPI/tAPI). Perhaps a comment would be good since the difference is hard to spot via ocular observation.